### ISSUE TITLE
error       Unable to resolve module `AccessibilityInfo`？

### ISSUE THREAD

**Issue by QAQXiYangYang** [facebook/react-native#20444]

**Comment by Tchanu** [comment]
Duplicate of #4968

**Comment by alexander0205** [comment]
That is the solution step by step 
`https://stackoverflow.com/questions/51333897/react-native-build-error-development-server-returned-response-error-code-500/51360535`
